backend/app/services/widgets/queries/summary.rb

```diff
@@ -0,0 +1,25 @@
+module Widgets
+  module Queries
+    class Summary < Base
+      private
+
+      def headers
+        [
+          {label: I18n.t("widgets.headers.summary.total_number_funders"), value: :total_funders},
+          {label: I18n.t("widgets.headers.summary.total_number_projects"), value: :total_projects},
+          {label: I18n.t("widgets.headers.summary.total_capital"), value: :total_capital},
+          {label: I18n.t("widgets.headers.summary.total_grants"), value: :total_grants}
+        ]
+      end
+
+      def data
```

Comment:
This looks great and I like the structure! You certainly thought this through and possibly you have a bigger pattern in mind, but if not then perhaps this could be called `values`, because otherwise it's "data in data" in the API response. But if Miguel is not bothered, neither am I :)